ansible-ipi-install/roles/installer/vars/main.yml

```diff
@@ -3,3 +3,16 @@
 pullsecret_file: "{{ ansible_user_dir }}/clusterconfigs/pull-secret.txt"
 cmd: openshift-baremetal-install
 default_libvirt_pool_dir: /var/lib/libvirt/images/
+
+# packages needed for the disconnected registry tasks
+packages_registry:
+  - podman
+  - libvirt
+  - python3
+  - jq
+  - httpd
+  - httpd-tools
+  - python3-dns
```

Comment:
need to reverify we need all these packages. I know libvirt, jq are already installed by default in our node-prep

Reply:
we are also making the assumption that if they use an outside registry host, that the server has a repo setup FYI

Reply:
you install some of these packages on the provisioning host. If the host for the disconnected is not the provisioning host, then they will need to be installed on it.